### DISCUSSION TITLE
[✨] Qwik City: navigating to current/same `url` using `<Link>` should got to top of the page.

### DISCUSSION THREAD

**Discussion by 0xElectric** [QwikDev/qwik-evolution#160] [Proposals For Qwik]
### Is your feature request related to a problem?

No

### Describe the solution you'd like

navigating to current/same `url` using `<Link>` should got to top of the page.

### Describe alternatives you've considered

I can add a logic in main layout to check if url is the same. But would be nice to have this on Qwik City by default.

**Comment by gioboa** [comment]
@Varixo is it implemented in v2?

**Comment by Varixo** [comment]
> @Varixo is it implemented in v2?

No, the Qwik City is not touched for now

**Comment by gioboa** [comment]
We moved this issue to `qwik-evolution` repo to create a RFC discussion for this.
Here is our [Qwik RFC process](https://github.com/QwikDev/qwik-evolution?tab=readme-ov-file#qwik-rfc-process) thanks.
